code/frameworks/angular/package.json

```diff
@@ -77,6 +77,7 @@
     "@angular/forms": "^15.1.1",
     "@angular/platform-browser": "^15.1.1",
     "@angular/platform-browser-dynamic": "^15.1.1",
+    "@ngxs/store": "^3.7.6",
```

Comment:
@valentinpalkovic Is this still needed here? I don't see any references to this package below. I assume this was added just for testing?

Reply:
Thank you! I will remove it again :)